### PR TITLE
[MINI-6242] Hotfix for access token for v5.3.1

### PR DESCRIPTION
**Pull request by munir-rakuten**
Allow host app to pass MiniAppManifest object using loadFromBundle method. Because the same MiniAppManifest is stored in Shared pref which will be used by the SDK for the scope cross-verification

# Checklist
- [x] I have read the [contributing guidelines](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md#sdk-development-learning-path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+- **Feature:** Added `MiniAppMessageBridge.didReceiveMAAnalytics` to receive analytics from MiniApp.
+- **Feature:** Added `MiniAppDisplay.sendInfoToHostApp` interface to provide `Universal Bridge` for sending messages from a MiniApp to the HostApp.
+
+**Sample App**
+- **Feature:** Added implementation of `MiniAppMessageBridge.getHostAppThemeColors`.
+- **Feature:** Added qa settings to test primary and secondary color.
+- **Feature:** Added Deeplink Support to change settings keys by QR code.
+- **Feature:** Added implementation of `MiniAppMessageBridge.getIsDarkMode`.
+- **Feature:** Added `MiniApp` button in Feature page to load a MiniApp from bundle.
+- **Feature:** Added `Logs` in `QA` page to show the analytics events come from miniapp.
```
Comment (rleojoseph) [inline CHANGELOG.md:24]: Candidate branch should already have this change. Why is this showing here 🤔

File: build.gradle [hunk 2]
```diff
@@ -97,10 +97,8 @@ allprojects {
         tag {
             prefix = 'v'
             versionSeparator = ''
-            versionIncrementer 'incrementMinor'
+            versionIncrementer 'incrementPatch'
         }
-        //override the version
-        versionCreator({versionFromTag,scmPosition -> "5.3.0-alpha"})
```
Comment (rleojoseph) [inline build.gradle:103]: Same comment as above

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewHandler.kt [hunk 3]
```diff
@@ -257,9 +257,22 @@ internal class MiniAppViewHandler(
...
+                miniAppDownloader.saveManifestForMiniAppBundle(
+                    appId = miniAppInfo.id,
+                    versionId = miniAppInfo.version.versionId,
+                    languageCode = "",
+                    manifest = manifest
+                )
+                downloadedManifestCache.storeDownloadedManifest(
+                    miniAppId = miniAppInfo.id,
+                    CachedManifest(miniAppInfo.id, manifest)
+                )
```
Comment (rleojoseph) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewHandler.kt:274]: Not sure why we store in two places, file also in preferences. Any reason ?

**Review by rleojoseph — COMMENTED** [review]
Thanks for the quick fix. I have added comments. Please check it

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
 ## CHANGELOG
 
+### 5.3.1 (XXXX-XX-XX)
+**SDK**
+- **Feature:** Added `manisfest` parameter to pass a manifest file while calling `MiniAppView.loadFromBundle`.
```
Comment (rleojoseph) [inline CHANGELOG.md:5]:
```suggestion
- **Feature:** Added `manifest` parameter to pass a `MiniAppManifest` object while calling `MiniAppView.loadFromBundle`.
```

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewHandler.kt (hunk 3 above)
Comment (munir-rakuten) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewHandler.kt:274]: yeah,i dont know why but it's like that in android, there are two file systems there and sometimes it check one and sometimes the other one, there are tickets for that change as well. MINI-2945

Comment (rleojoseph) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewHandler.kt:274]: Wow. Nice, so we need to get rid of SharedPref approach soon. Thanks @munir-rakuten . We can have this for now.

File: CHANGELOG.md (hunk 1 above)
Comment (rleojoseph) [reply CHANGELOG.md:24]: This is strange

**Review by rleojoseph — APPROVED** [review]

**Review by GondiTilak — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**